web/styles/dark_theme.css

```diff
@@ -376,11 +368,6 @@
         border-color: hsl(0deg 0% 0% / 60%);
     }
 
-    .table-striped thead th {
```

Comment:
Because of their low specificity, these styles were unreachable. Screenshots further show no ill effects from this removal.